experimental/trusted_shuffler/server/src/http.rs

```diff
@@ -63,30 +74,23 @@ impl hyper::service::Service<Request<Body>> for Service {
     }
 
     fn call(&mut self, request: Request<Body>) -> Self::Future {
+        log::info!("Received Request: {:?}", request);
         let trusted_shuffler = self.trusted_shuffler.clone();
         let response = async move {
-            match (request.method(), request.uri().path()) {
-                (&Method::POST, "/request") => {
-                    let body = hyper::body::to_bytes(request.into_body())
-                        .await
-                        .expect("Couldn't read request body");
-
+            match request.method() {
+                // TODO(mschett): Check if http still works.
+                // (&Method::POST, "/experimental.trusted_shuffler.echo.Echo/Echo")
+                // (&Method::POST, "/request")
+                &Method::POST => {
```

Comment:
I think we should accept any requests, not just `POST` ones

Reply:
Done!